### ISSUE TITLE
Separate fleet & ship detail scrolling in outfitter

### ISSUE THREAD

**Issue by SubstandardZeal** [endless-sky/endless-sky#5143]
**Is your feature request related to a problem? Please describe.**
I often find myself scrolling between the fleet panel and ship details near the bottom like energy & heat.

**Related Issue Links**
I have looked through these issues but no one has suggested it yet: https://github.com/endless-sky/endless-sky/pull/3533 https://github.com/endless-sky/endless-sky/issues/3549

**Describe the solution you'd like**
I suggest the fleet and ship detail panels have seperate scroll bars. This would allow users to keep focus on a section of the fleet and specific ship details at the same time.

Perhaps the fleet panel could grow as it currently does to some sensible limit. Perhaps also users could drag the "divider" between the panels to their own preference or perhaps a third scrollbar might scroll both as the game currently does.

**Mockup:**
![Annotation 2020-07-02 133853](https://user-images.githubusercontent.com/21998611/86409840-2e452600-bca9-11ea-952c-8f19b1a05c15.png)

**Comment by MasterOfGrey** [comment]
I think in this case the divider is unnecessary and the “Regular Scrollbar” should simply have the effect of moving between 2/3 fleet +1/3 ship and 1/3 fleet +2/3 ship.

**Comment by Amazinite** [comment]
I think this is basically a duplicate of #3326.

**Comment by MasterOfGrey** [comment]
Yes it is essentially a duplicate, just with a *superior* solution to the core issue.

**Comment by xobes** [comment]
If the direction is agreed upon, I can take a crack at implementing it...

**Comment by Amazinite** [comment]
Go for it. I'm still pretty fond of the side-by-side mockup I had in #3326, but I understand that it's not the best approach for narrower screens, especially now since we've moved the selected ship info to its own column instead of being underneath the selected item. (Part of me says implement both and use a preference and/or the screen width to determine which interface to display, but that might be overkill.)

**Comment by jwmh** [comment]
> (...), but I understand that it's not the best approach for narrower screens,
> especially now since we've moved the selected ship info to its own column instead of being underneath the selected item. (Part of me says 
> **implement both and use a preference** (...)
> to determine **which interface to display**, but that might be overkill.)

I wholly support preferences to allow the user to choose their display interface.

Would this be hard to do?
